process_report/processors/lenovo_processor.py

```diff
@@ -0,0 +1,14 @@
+from dataclasses import dataclass
+
+
+from process_report.invoices import invoice
+from process_report.processors import processor
+
+
+@dataclass
+class LenovoProcessor(processor.Processor):
+    SU_CHARGE_MULTIPLIER = 1
+
+    def _process(self):
+        self.data["SU Charge"] = self.SU_CHARGE_MULTIPLIER
```

Comment:
Declare these strings as columns somewhere, I think for now we're keeping all of them in the invoice base module?